app/models/cv_validation_job.rb

```diff
@@ -50,21 +54,24 @@ class CvValidationJob
   #   @return [Array<String>] The messages that were logged during the CV Validation Job. This is a placeholder for future enhancements/use.
   field :logging_messages, type: Array
 
-  # @!attribute [rw] cv_start_time
-  #   @return [DateTime] The start timestamp to track how long the CV payload took to generate.
-  field :cv_start_time, type: DateTime
+  # @!attribute [rw] cv_payload_transformation_time
+  #   @return [BigDecimal] The time taken to transform the Family into a valid Aca Entity.
+  field :cv_payload_transformation_time, type: BigDecimal
 
-  # @!attribute [rw] cv_end_time
-  #   @return [DateTime] The end timestamp to track how long the CV payload took to generate.
-  field :cv_end_time, type: DateTime
+  # @!attribute [rw] job_elapsed_time
+  #   @return [BigDecimal] The time taken to complete the CV Validation Job.
+  field :job_elapsed_time, type: BigDecimal
 
-  # @!attribute [rw] start_time
-  #   @return [DateTime] The start timestamp of the CV Validation Job.
-  field :start_time, type: DateTime
+  # Validations
 
-  # @!attribute [rw] end_time
-  #   @return [DateTime] The end timestamp of the CV Validation Job.
-  field :end_time, type: DateTime
+  # @!attribute [rw] result
```

Comment:
Small note-- I noticed the documentation on the `:result` field on line 47 above needs to be updated to include `:error` as a possible value.